test/cmd/newapp.sh

```diff
@@ -9,10 +9,18 @@ source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/cmd_util.sh"
 os::log::install_errexit
 
+# Cleanup cluster resources created by this test
+(
+  set +e
+  oc delete all,templates --all
+  exit 0
+) &>/dev/null
```

Comment:
> But you could also accidentally delete things by running
> oc delete all --all in normal use,

I've hit this while working on the project :smile: . The catch here is that whenever the development server is restarted, it sends you back to the `default` namespace. https://github.com/openshift/origin/issues/2516

I "learned" to always include a "-n" argument in my `oc delete` calls.


> or run rm -rf *

I wouldn't like a project that deletes my files when I run the test suite. This reminds of the epic [bumblebee](https://github.com/MrMEEE/bumblebee-Old-and-abbandoned/commit/a047be85247755cdbe0acce6f1dafc8beb84f2ac#diff-3fbb47e318cd8802bd325e7da9aaabe8L351).


I wish there were no "risks" involved when running tests.
IMHO, it's easier to tolerate test suites that leaves behind temporary files (as long as it doesn't make your disk bleed) than having files changed or removed.